specs/waku/v2/waku-relay.md

```diff
@@ -22,11 +30,34 @@ authors: Oskar Thorén <oskar@status.im>
 
 **Protocol identifier***: `/vac/waku/relay/2.0.0-beta2`
 
+## Security Requirements
```

Comment:
I wonder if this is "security requirements" or if something like "design goals" would make sense. How do you think about this distinction? Aspirational/design vs requirements <for something>

Reply:
I agree, "security requirements" might be misleading as the reader may think these are addressed in the protocol, but I feel the same about the "design goals", it sounds like the current design covers all the listed security features (which is not the case). The word "Design" is also general and includes other aspects like performance as well. What about "Security Aspects"?